frontend/controller/serviceworkers/sw-primary.js

```diff
@@ -104,7 +104,7 @@ sbp('okTurtles.events/on', CHELONIA_RESET, setupRootState)
 
 // These are all of the events that will be forwarded to all open tabs and windows
 ;[
-  CHELONIA_RESET, CONTRACTS_MODIFIED, CONTRACT_IS_SYNCING, CONTRACT_REGISTERED,
+  CHELONIA_RESET, CONTRACTS_MODIFIED, CONTRACT_IS_SYNCING,
```

Comment:
Why is this removing CONTRACT_REGISTERED? Didn't we just add that at Sebin's request?